frontend/controller/actions/identity.js

```diff
@@ -326,7 +326,9 @@ export default (sbp('sbp/selectors/register', {
       // NOTE: users could notice that they leave the group by someone else when they log in
       if (!state.currentGroupId) {
         const { contracts } = state
-        const gId = Object.keys(contracts).find(cID => contracts[cID].type === 'gi.contracts/group')
+        const gId = Object.keys(contracts)
```

Comment:
Since it's unclear what line 331 is doing, could you please add this comment above line 329?

```js
// grab the groupID of any group that we've successfully finished joining
```